aea/cli/run.py

```diff
@@ -85,17 +83,13 @@ def run(ctx: Context, connection_name: str):
     """Run the agent."""
     _try_to_load_agent_config(ctx)
     agent_name = cast(str, ctx.agent_config.agent_name)
-    private_key_pem_path = cast(str, ctx.agent_config.private_key_pem_path)
-    if private_key_pem_path == "":
-        private_key_pem_path = _create_temporary_private_key_pem_path()
-    else:
-        _try_validate_private_key_pem_path(private_key_pem_path)
-    crypto = Crypto(private_key_pem_path=private_key_pem_path)
-    public_key = crypto.public_key
+    wallet = Wallet()
```

Comment:
I think you need to pass the private key paths?

Reply:
Not anymore. I am loading them from the aea-config.yaml